### ISSUE TITLE
Tracking: Missing rules in selection reasons in dependencyInsight

### ISSUE THREAD

**Issue by OdysseusLives** [nebula-plugins/gradle-nebula-integration#41]
Tracking issue gradle/gradle#6826

**Comment by bigdaz** [comment]
This has been fixed via https://github.com/gradle/gradle/pull/7582

Reasons for all rules that influence resolution are now collected and reported in `dependencyInsight`.

**Comment by OdysseusLives** [comment]
Cool! While you're collecting reasons, are you also collecting information from `useVersion`?

**Comment by bigdaz** [comment]
@OdysseusLives No, that will be a bigger change and will likely have a significant performance impact. I'll try to find some time to spike the solution this week.

**Comment by OdysseusLives** [comment]
Good to know. Thanks for the update!
